java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -22,6 +22,8 @@ public class TransactionTestUtilities {
     private static final String key7 = "{key}" + UUID.randomUUID();
     private static final String key8 = "{key}" + UUID.randomUUID();
     private static final String key9 = "{key}" + UUID.randomUUID();
+    private static final String key10 = "{key}" + UUID.randomUUID();
```

Comment:
I recommend naming new key like `zSetKey2` and avoid changing existing tests

Reply:
Like this?